app/assets/stylesheets/modules/top_nav.scss

```diff
@@ -27,6 +39,8 @@
       font-size: 0.9em;
       margin-right: 1.5em;
       padding-bottom: 0.3em;
+      font-weight: bold;
+      color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.